apps/nuxt3-ssr/interfaces/types.ts

```diff
@@ -197,3 +207,58 @@ export enum INotificationType {
   warning,
   info,
 }
+
+export interface ILocale {
+  locale: string;
```

Comment:
no strong opinion on this , no a sure the current use of locale justifies the current added complexity ( this would argue in  favour of moving into backend )

Reply:
indeed.